.gitignore

```diff
@@ -1,3 +1,27 @@
 target/
```

Comment:
nit: I would split this off into a separate PR since it's not related to the emoji fix.